README.md

```diff
@@ -24,4 +24,31 @@ poeditor: {
   }
 ```
 
+## Download all languages
+Another task is to download all languages, with minimum poeditor coverage.
+
+```
+poeditor: {
+  download_all: {
+    // Same configuration as download
+    download_all: {
+      dest: 'tests/?.json',
+
+      // Specify minimum coverage here
+      minimum_coverage: 40
+    }
+  },
```

Comment:
the first is the name of the task, the second is real grunt task.
we have `download.download` in our gruntfile under `poeditor`.